changelog/add-8557-empty-state-payment-widget

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Add an empty state view for the Payments Activity widget. This is shown when the merchant is yet to have any transacitons over WooPayments.
```

Comment:
Spelling mistake here. 
Also, as this is behind a feature flag and part of a larger set of features it doesn't need to be on a userfacing changelog.
```suggestion
Significance: patch
Type: add
Comment: Add an empty state view for the Payments Activity widget. This is shown when the merchant is yet to have any transactions over WooPayments.
```

Reply:
Done! - 34e125b